src/devices/acurite.c

```diff
@@ -592,15 +661,15 @@ static int acurite_txr_decode(r_device *decoder, bitbuffer_t *bitbuffer)
                         "battery",      NULL,   DATA_STRING,    battery_low ? "OK" : "LOW",
                         _X("wind_avg_km_h","wind_speed_kph"),   "wind_speed",   DATA_FORMAT,    "%.1f km/h", DATA_DOUBLE,     wind_speed_kph,
                         "temperature_F",     "temperature",    DATA_FORMAT,    "%.1f F", DATA_DOUBLE,    tempf,
-                        "humidity",     NULL,    DATA_FORMAT,    "%d",   DATA_INT,   humidity,
+                        "humidity",     NULL,    DATA_FORMAT,    "%u %%",   DATA_INT,   humidity,
```

Comment:
See comment above, would prefer not to have the Atlas PR change the output format for other devices